packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -216,6 +216,25 @@ export class ListItem
     SelectionMode
   > = null;
 
+  /**
+   * Specifies the interaction mode of the component - `"interactive"` (allows interaction styling and pointer changes on hover), `"static"` (does not allow interaction styling and pointer changes on hover), Do not combine `interaction-mode=“static”` and `selection-appearance=“border”`.
+   * @internal
+   */
+  @Prop({ reflect: true }) interactionMode: InteractionMode = null;
+
+  @Watch("interactionMode")
+  @Watch("selectionAppearance")
+  @Watch("selectionMode")
+  handleInteractionModeSelectionAppearanceSelectionModeChange(): void {
```

Comment:
I think this would need to be called on `componentWillLoad` as well, otherwise it would only warn after one of these was changed.

Reply:
Should this warning occur on the list instead of the list item? The property on the list-item is internal so its weird to log here.

Reply:
I moved this logic into list.tsx 🚀